engine/table/src/test/java/io/deephaven/engine/table/impl/FuzzerTest.java

```diff
@@ -117,7 +117,7 @@ private void testFuzzerScriptFile(final long timeSeed, String s, boolean realtim
             clock.now += DateTimeUtils.SECOND / 10 * timeRandom.nextInt(20);
         }
 
-        final TimeTable timeTable = (TimeTable) session.getVariable("tt");
+        final TimeTable timeTable = session.getQueryScope().readParamValue("tt", null);
```

Comment:
Won't this NPE if it doesn't exist? Probably you meant to leave this as throwing a MissingVarException?

Reply:
You're not wrong, but the test will fail either way.